### ISSUE TITLE
[Doc]: Wrongly spelled option in HTML

### ISSUE THREAD

**Issue by gruberma** [matplotlib/matplotlib#20861]
### Documentation Link

https://matplotlib.org/2.1.0/users/whats_new.html#reproducible-ps-pdf-and-svg-output

### Problem

Concerning the paragraph

> Alternatively, calling savefig with metadata={'creationDate': None} will omit the timestamp altogether for the PDF backend.

* On the Website, `creationDate` is spelled with a lower "c".
* This is wrong, the real name of the option is `CreationDate`
* In the [respective RST file](https://github.com/matplotlib/matplotlib/blob/96657657e9adaabf9d61d590aee7a43b0e2c1d2f/doc/users/prev_whats_new/whats_new_2.1.0.rst#reproducible-ps-pdf-and-svg-output), however, the spelling is correct.

------------------------------------------------

Sidenotes about the behavior of this option:
When providing the wrongly spelled option (`creationDate`), I only get a warning, if the value of the option in not None:

```python
plt.plot([1,2,4]); plt.savefig("foo.pdf", metadata={'creationDate': 1629363355}); plt.show()
```
results in
```
<ipython-input-4-3f7daa9b46e5>:1: UserWarning: Unknown infodict keyword: 'creationDate'. Must be one of {'Author', 'Keywords', 'Trapped', 'Creator', 'ModDate', 'Title', 'Subject', 'CreationDate', 'Producer'}.
  plt.plot([1,2,4]); plt.savefig("foo.pdf", metadata={'creationDate': 1629363355}); plt.show()
```

While
```python
plt.plot([1,2,4]); plt.savefig("foo.pdf", metadata={'creationDate': None}); plt.show()
```
does not result in a warning

### Suggested improvement

1. Change spelling on website
2. Maybe also give a warning if the value is `None` (this is not related to documentation though)

### Matplotlib Version

3.4.1

### Matplotlib documentation version

 2.1.1

**Comment by timhoffm** [comment]
Thanks for the report. https://matplotlib.org/2.1.0/ is the docs as released for 2.1.0. The typo has been fixed after the release in the .rst file. We do not regenerate docs for technical and practical reasons.

The fix is active in the 
current docs: https://matplotlib.org/stable/users/prev_whats_new/whats_new_2.1.0.html

**Comment by gruberma** [comment]
perfect, thanks for the explanation 👍
